origin-dapp/src/utils/listing.js

```diff
@@ -62,6 +61,27 @@ export function dappFormDataToOriginListing(formData) {
   return listingData
 }
 
+function getCategory(category) {
```

Comment:
nit: for both methods, add a comment explaining why we are doing this (for backward compatibility purposes to support legacy listings created in now deprecated cats/subcats).